rest-api/tools/consent_upload_validation.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -ae
+
+# Read participant ID's from csv and compare consent docs from ptc-upload-all-of-us-rdr-prod to a given
+# source bucket (i.e. aouxxx). Creates a new spreadsheet with discrepancies.
```

Comment:
The source bucket here, I don't remember where this came from. Is this something that exists in prod? Who uploads it and what do we expect to find in the bucket?

Reply:
added more comments. and yes only in prod.